### PR TITLE
Pic 3530 - move to aws secrets

### PR DESCRIPTION
**Pull request by johnyfarrar**

### CODE REVIEW

File: helm_deploy/prepare-a-case/templates/_envs.tpl [hunk 1]
```diff
@@ -112,5 +112,17 @@ env:
...
+    valueFrom:
+      secretKeyRef:
+        name: liverpool-pre-pilot-users
+        key: liverpool_pre_pilot_users
+
+  - name: CASE_TRACKING_PRE_PILOT_USERS
+    valueFrom:
+      secretKeyRef:
+        name: case-tracking-pre-pilot-users
+        key: case_tracking_pre_pilot_users
```
Comment (markreesmoj) [inline helm_deploy/prepare-a-case/templates/_envs.tpl:126]: recommend moving to use the helm generic service ASAP in line with (most of) our other projects

Comment (markreesmoj) [reply helm_deploy/prepare-a-case/templates/_envs.tpl:126]: also pre-pilot :-)

**Review by markreesmoj — APPROVED** [review]
👍

File: helm_deploy/prepare-a-case/templates/_envs.tpl (hunk 1 above)
Comment (johnyfarrar) [reply helm_deploy/prepare-a-case/templates/_envs.tpl:126]: Is that the key? Do I have a typo?

Comment (markreesmoj) [reply helm_deploy/prepare-a-case/templates/_envs.tpl:126]: no I just found the notion of a pre-pilot phase amusing.

**Pull request merged**